### PR TITLE
Added ReviewNinja badge

### PR DESCRIPTION
**Pull request by valgog**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/30" target="_blank">ReviewNinja

**Comment by valgog** [comment]
Let's see how the review ninja is _sneaking_ around

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-[![Build Status](https://travis-ci.org/zalando/nakadi.svg)](https://travis-ci.org/zalando/nakadi) [![Swagger API](http://online.swagger.io/validator?url=https://raw.githubusercontent.com/zalando/nakadi/master/nakadi/swagger.yaml)](http://online.swagger.io/validator/debug?url=https://raw.githubusercontent.com/zalando/nakadi/master/nakadi/swagger.yaml)
+[![Build Status](https://travis-ci.org/zalando/nakadi.svg)](https://travis-ci.org/zalando/nakadi) [![ReviewNinja](https://app.review.ninja/44234368/badge)](https://app.review.ninja/zalando/nakadi)
+
+[![Swagger API](http://online.swagger.io/validator?url=https://raw.githubusercontent.com/zalando/nakadi/master/nakadi/swagger.yaml)](http://online.swagger.io/validator/debug?url=https://raw.githubusercontent.com/zalando/nakadi/master/nakadi/swagger.yaml)
```
Comment (valgog) [inline README.md]: Strange place to leave comments

Comment (valgog) [inline README.md]: And again, it is not possible to write a hierarchy of comments :( !completed

Comment (valgog) [inline README.md]: Actually comments in ReviewNinja are directly translated to the comments on the pull requests :D that is actually good

Comment (v-stepanov) [inline README.md]: I suggest to lower the required amount of ninja stars to 1.
Because sometimes it is difficult to get even one approval.

Comment (valgog) [inline README.md]: One is from you yourself 😀

Comment (v-stepanov) [inline README.md]: Come on, approving your own PR is similar to liking your own post on facebook :)

**Pull request merged**